src/PointwiseFunctions/AnalyticData/CurvedWaveEquation/PureSphericalHarmonic.hpp

```diff
@@ -91,14 +81,14 @@ class PureSphericalHarmonic : public MarkAsAnalyticData {
             tmpl::list<CurvedScalarWave::Pi, CurvedScalarWave::Phi<3>,
                        CurvedScalarWave::Psi> /*meta*/) const noexcept;
 
-  // clang-tidy: no runtime references
+  // NOLINTNEXTLINE(google - runtime - references)
```

Comment:
If clang-tidy accepts this, fine. Else delete spaces around dashes.

Delete the `NOLINT` in the line below.

Reply:
you haven't deleted the `NOLINT` in the line below